### DISCUSSION TITLE
Update database curso Full Stack

### DISCUSSION THREAD

**Discussion by lucasmiko** [balta-io/forum#584] [Tira Dúvidas] [answered]
Boa tarde,

Na aula Entity Framework > Migrations, não estou conseguindo executar o update database.

Realizei o clean, build, restore... O que que estou é esse:

![image](https://github.com/balta-io/forum/assets/63825231/a70b5836-07e8-4eca-ac6b-b2e31879b22d)

![image](https://github.com/balta-io/forum/assets/63825231/0f5c9f8a-5cdb-4c21-81ae-f38386f35ba7)

Tentei setar o EnableRetryOnFailure mas o erro persistiu ainda assim.

**Comment by lucasmiko** [comment]
.

**Reply by andrebaltieri** [reply]
?

**Comment by andrebaltieri** [comment, marked as answer]
![Screenshot 2024-06-15 at 10 50 13](https://github.com/balta-io/forum/assets/965305/c59a9b92-c7e8-4911-a2e6-860bcb48ab21)

Duplicado
https://github.com/balta-io/forum/discussions/139
